README.md

```diff
@@ -37,6 +38,30 @@ Import utils like so:
 import { URN } from "@scaife-viewer/scaife-widgets";
 ```
 
+Import and initialize the store like so:
+
+```js
+import Vue from "vue";
+import Vuex from "vuex";
+import App from "./App.vue";
+import { scaifeWidgets } from "@scaife-viewer/scaife-widgets";
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  modules: {
+    scaifeWidgets
```

Comment:
Is this shorthand equivalent to [this](https://github.com/scaife-viewer/explorehomer/blob/scaife/widgets/src/store.js#L20)?

Reply:
yes

Reply:
cool! didn't see it done that way anywhere else, handy.

Reply:
Well, it only works when the module name is the same as the variable - `scaifeWidgets`.

Reply:
ah, if the `namespace` name is the same? i get you, it's just regular object shorthand. still a nice shortcut.

Reply:
Version 1:

```js
const scaifeWidgets = {
  namespaced: true,
  state,
  actions,
  getters,
  mutations
};

const store = new Vuex.Store({
  modules: { scaifeWidgets }
});
```

Version 2:

```js
const scaifeWidgets = {
  namespace: MODULE_NS,
  store: {
    namespaced: true,
    state,
    actions,
    getters,
    mutations
  }
};

...

const store = new Vuex.Store({
  modules: {
    [scaifeWidgets.namespace]: scaifeWidgets.store
  }
});
```

Version 3:

```js
const scaifeWidgets = {
  namespace: MODULE_NS,
  store: {
    namespaced: true,
    state,
    actions,
    getters,
    mutations
  }
};

const module = {};
module[scaifeWidgets.namespace] = scaifeWidgets.store
const store = new Vuex.Store({
  modules: module
});
```

Reply:
Depends on how readable you like your code...

Reply:
#2 is what I see most. I find it hard to read `[scaifeWidgets.namespace]`, though...

Reply:
right, so you are eliding the `store` to just splat it all in via the matching names.

I guess #2 is the most common, as you say, as it's the way it's done in the Vue docs. Ultimately I think we just need to be consistent and I think `scaife-skeleton` follows #2, so we'd have to change the shape of that module's data too if we go with anything different.